src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const styleObject = {};
+  const styleCommands = sourceString.split(';');
+
+  styleCommands.forEach((element) => {
+    const [key, value] = element.split(':');
+
+    if (key && value) {
+      styleObject[key.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Instead of using `forEach` to populate `styleObject`, consider using `reduce` which is more suited for transforming an array into a single value or object. This aligns with the checklist item to avoid using loops and prefer iteration methods.